types/openebs.go

```diff
@@ -299,14 +299,13 @@ type JivaConfig struct {
 // pool, poolMgmt, target and volumeMgmt are the containers which are deployed
 // in the k8s cluster.
 type CstorConfig struct {
-	Pool          Container     `json:"pool"`
-	PoolMgmt      Container     `json:"poolMgmt"`
-	Target        Container     `json:"target"`
-	VolumeMgmt    Container     `json:"volumeMgmt"`
-	CSPIMgmt      Container     `json:"cspiMgmt"`
-	VolumeManager Container     `json:"volumeManager"`
-	CSPCOperator  *CSPCOperator `json:"cspcOperator"`
-	CVCOperator   *CVCOperator  `json:"cvcOperator"`
+	Pool         Container     `json:"pool"`
```

Comment:
yes @shovanmaity , we have only one exporter image i.e.,quay.io/openebs/m-exporter:1.8.0 and it can be configured via
```
policies:
    monitoring:
      enabled:
      imageTag:

```